crates/viewer/re_viewer_context/src/viewer_context.rs

```diff
@@ -7,17 +7,19 @@ use re_entity_db::entity_db::EntityDb;
 use re_query::StorageEngineReadGuard;
 
 use crate::drag_and_drop::DragAndDropPayload;
+use crate::GlobalContext;
 use crate::{
     query_context::DataQueryResult, AppOptions, ApplicationSelectionState, CommandSender,
     ComponentUiRegistry, DragAndDropManager, IndicatedEntities, ItemCollection,
     MaybeVisualizableEntities, PerVisualizer, StoreContext, SystemCommandSender as _, TimeControl,
     ViewClassRegistry, ViewId,
 };
 
-/// Common things needed by many parts of the viewer.
-pub struct ViewerContext<'a> {
+impl ViewerContext<'_> {
```

Comment:
nit: I appreciate that this keeps the diff smaller, but it's a bit strange to first have the impl and then the struct